circus-train-s3-mapreduce-cp-copier/src/main/java/com/hotels/bdp/circustrain/s3mapreducecpcopier/S3MapReduceCpCopier.java

```diff
@@ -164,6 +164,14 @@ public Long getValue() {
 
   }
 
+  private void loadDistributedFileSystems() throws IOException {
+    new LibJarDeployer().libjars(conf,
+        org.apache.hadoop.fs.s3a.S3AFileSystem.class,
+        com.hotels.bdp.circustrain.aws.BindS3AFileSystem.class,
+        com.google.cloud.hadoop.fs.gcs.GoogleHadoopFileSystem.class,
```

Comment:
Can we extend this copier in our plugin and add these classes on demand?

Reply:
As you notice here, this is the reason that the copier is depending on the gcp module in order to get the GCP classes on the classpath so that the Hadoop FileSystem API can understand how to access them. @courtsVII can something like the above be done in the GCP module? Maybe on startup or before calling this copier somehow?

Reply:
Changed to do this bit of configuration in respective BindFileSystem implementations of circus-train-gcp and circus-train-aws